builder/testdata/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM quay.io/kiegroup/kogito-serverless-workflow-base:latest AS builder
+#Not yet released Change the 31 st oct with quay.io/kiegroup/kogito-serverless-workflow-base:latest
+FROM quay.io/mdessi/kogito-serverless-workflow-base:latest AS builder
```

Comment:
don't we have it in the kiegroup repo  yet?
if not, can you please push it?

Reply:
@spolti We have a nightly version https://quay.io/repository/kiegroup/kogito-swf-builder-nightly

Reply:
can we use this one?

Reply:
or push a snahpshot?
If we can use a snapshot, i can push it.

Reply:
Fixed with "FROM quay.io/repository/kiegroup/kogito-swf-builder-nightly:latest AS builder" until 31 oct with the official version